apps/voting/app/src/components/DisputableStatusLabel.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import {
+  GU,
+  IconAttention,
+  IconClock,
+  IconClosed,
+  IconInfo,
+  Tag,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import {
+  VOTE_STATUS_PAUSED,
+  VOTE_STATUS_ACTIVE,
+  VOTE_STATUS_CANCELLED,
+  VOTE_STATUS_CLOSED,
+} from '../disputable-vote-statuses'
+import { getDisputableVoteStatus } from '../disputable-utils'
+
+const getStatusAttributes = (status, theme) => {
+  if (status === VOTE_STATUS_PAUSED) {
+    return {
+      background: theme.warningSurface,
+      label: 'Challenged',
+      Icon: IconAttention,
+      color: theme.warningSurfaceContent,
+    }
+  }
+  if (status === VOTE_STATUS_CANCELLED) {
+    return {
+      background: theme.surfaceUnder,
+      label: 'Cancelled',
+      Icon: IconClosed,
+      color: theme.disabledContent,
+    }
+  }
+  if (status === VOTE_STATUS_CLOSED) {
+    return {
+      background: theme.surfaceUnder,
+      label: 'Closed',
+      Icon: IconInfo,
+      color: theme.disabledContent,
+    }
+  }
+
+  if (status === VOTE_STATUS_ACTIVE) {
+    return {
+      background: null,
+      label: 'Scheduled',
+      Icon: IconClock,
+      color: null,
+    }
+  }
+  return {
+    background: null,
+    label: '',
+    Icon: null,
+    color: null,
+  }
+}
+
+function DisputableStatusLabel({ vote }) {
+  const theme = useTheme()
+  const status = getDisputableVoteStatus(vote)
```

Comment:
As mentioned, we could only accept `status` as the prop as it's all we need to determine the style of our `Tag`. Also be sure to add propTypes for the 4 that it accepts.